pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>io.github.jaredpetersen</groupId>
   <artifactId>kafka-connect-arangodb</artifactId>
-  <version>1.0.6</version>
+  <version>1.0.7</version>
```

Comment:
I wouldn't bump this up without a sem-var build or pre-release modifier. Since it's not an official release.

Reply:
https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions

Reply:
I did this for the official release, changeset was merged a while ago into the original project.